packages/extension/babel.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = function (api) {
```

Comment:
Though Currently, we only have two packages, namely `cli` and `extension`, in the packages folder. However, we created this folder with the intention of potentially adding more packages in the future. Therefore, I believe it would be ideal to have these configuration files in the root folder instead. By doing so, we can ensure that they are shared across all packages.

Reply:
100%. We will sync on recent discussions, but related to this: let's follow closely a system's approach where the tool's architecture is, while being a prototype, extensible, maintainable, easy to understand. This applies to the build system, and a modular architecture for the tool. 

The "easy to understand" is key as this tool is all about privacy, and analyzing scenarios where 3P cookies play a role, and as an open source project, it should be easy for any one to clearly assess what the tool does exactly, what information is collected, and what is the flow of handling that information. 

Related to [Issue #11](https://github.com/GoogleChromeLabs/cookie-analysis-tool/issues/11).